geozero/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.13.0 - (2024-05-XX)
+
+* Fixed converting 2D geos::Geometry to ewkt
+  * <https://github.com/georust/geozero/pull/203>
+* Implement `srid` for GeosWriter
+  * <https://github.com/georust/geozero/pull/201>
+* Update docs with requisite `feature` requirements
+  * <https://github.com/georust/geozero/pull/205>
+* Omit GeoJSON properties with null values
+  * <https://github.com/georust/geozero/pull/206>
+* FIX: `with-gdal` feature missing `gdal-sys` requirement (broken in v0.12.0)
+  *  <https://github.com/georust/geozero/pull/212>
+* BREAKING: GeoJSONReader decodes `Array` and `Objects` as type `JSON`, not `String`
+            GeoJSONWriter writes `JSON` props, rather than ignoring them.
```

Comment:
```suggestion
* BREAKING: 
  * GeoJSONReader decodes `Array` and `Objects` as type `JSON`, not `String`
  * GeoJSONWriter writes `JSON` props, rather than ignoring them.
```
slightly more readable to have the second level bullets?